dev/Deployment/DeploymentManager.cpp

```diff
@@ -332,23 +332,17 @@ namespace winrt::Microsoft::Windows::ApplicationModel::WindowsAppRuntime::implem
         THROW_IF_WIN32_BOOL_FALSE(InitializeProcThreadAttributeList(attributeList, attributeCount, 0, &attributeListSize));
         auto freeAttributeList{ wil::scope_exit([&] { DeleteProcThreadAttributeList(attributeList); }) };
 
-        // Desktop Bridge applications by default have their child processes break away from the parent process.
-        // In order to recreate the calling process' environment correctly, we must prevent child breakaway semantics
-        // when calling the agent. Additionally the agent must do the same when restarting the caller.
-        DWORD policy{ PROCESS_CREATION_DESKTOP_APP_BREAKAWAY_OVERRIDE };
+        // The process being created will create any child processes outside of the desktop app runtime environment.
+        // This behavior is the default for processes for which no policy has been set.
+        DWORD policy{ PROCESS_CREATION_DESKTOP_APP_BREAKAWAY_ENABLE_PROCESS_TREE };
```

Comment:
Please add to the comment the link to the docs https://docs.microsoft.com/en-us/windows/win32/api/processthreadsapi/nf-processthreadsapi-updateprocthreadattribute

>PROCESS_CREATION_DESKTOP_APP_BREAKAWAY_OVERRIDE 0x04
>The process being created will run inside the desktop app runtime environment. This policy applies only to the process being created, not its descendants..

>PROCESS_CREATION_DESKTOP_APP_BREAKAWAY_ENABLE_PROCESS_TREE 0x01
>The process being created will create any child processes outside of the desktop app runtime environment. 

DeploymentAgent is derived from RestartAgent so I'm surprised this is a problem but perhaps RestartAgent has subtly different needs. RestartAgent uses the Override option but that's clearly not what we want here. The ProcessTree option gets us out of the packaged context as we want. The naming's confusing (Override doesn't, but ProcessTree does? Gah!)

DeploymentAgent's process token has the WIN://SYSAPPID + WIN://PKG access claim with the Override option but not with the ProcessTree option?

Reply:
Added the docs link.